builders/tools/explorers/verify-contract.md

```diff
@@ -0,0 +1,102 @@
+
+We will use the same contract as the above deployment tutorial for our contract verification example. 
+
+The contract used is a simple incrementer, arbitrarily named _Incrementer.sol_, which you can find [here](/snippets/code/web3-contract-local/Incrementer.sol). The Solidity code is the following:
+
+```solidity
+--8<-- 'code/web3-contract-local/Incrementer.sol'
+```
+
+### Collecting Information for Contract Verification
+
+You will need to collect some information related to the contract's compiler and deployment in order to verify it successfully. 
+
+1. Take note of the Solidity compiler version used to compile and deploy the contract. The Solidity compiler version can usually be selected or specified in the deployment tool used
+2. Take note of any SPDX license identifier used at the beginning of the Solidity source file (the example uses MIT license):
+    ```
+    // SPDX-License-Identifier: MIT
+    ```
+3. (Optional) If optimization is enabled during compilation, take note of the value of the optimization runs parameter
+4. (Optional) If the contract constructor method accepts arguments, take note of the [ABI-encoded form](https://docs.soliditylang.org/en/develop/abi-spec.html) of the constructor arguments
+5. After deployment, take note the deployed contract address of the smart contract. The deployment address of the contract can be found either in the console output if using a command line based tool such as Truffle, Hardhat, or an Ethereum library, or it can be copied from the GUI in tools such as Remix IDE
+
+![Example Compiler Options in Remix IDE](/images/builders/tools/explorers/verify-contract/verify-contract-1.png)
+
+## Verify the Contract  {: #verifying-the-contract }
+
+The next step will be verifying the smart contract in an EVM compatible explorer for the Moonbeam network that you deployed to. 
+
+### Verify on Moonscan {: #verify-on-moonscan }
+
+Take the following steps to verify the contract on Moonscan: 
+
+1. Go to the [Verify & Publish Contract Source Code](https://moonbase.moonscan.io/verifyContract) page of Moonscan
+2. Fill in the contract's deployed address in the first field including the `0x` prefix
+3. Select the compiler type. For the current `Incrementer.sol` example, select **Solidity (Single file)**
+4. After selecting the compiler type, select the compiler version used to compile the contract. If the compiler version used was a nightly commit, uncheck the box under the field to select the nightly version
+5. Select the open source license used. If there was none used, select **No License (None)**
+6. Click the **Continue** button at the bottom of the form to continue on to the next page
+    ![First Page Screenshot](/images/builders/tools/explorers/verify-contract/verify-contract-2.png)
```

Comment:
I suggest using the same sort of format that we normally do. Full window screenshot with arrows and numbers pointing each step to the numbered list.